new: extend SEND_X and SENDTO_X

ekoops opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines in the [CONTRIBUTING.md](https://github.com/falcosecurity/.github/blob/master/CONTRIBUTING.md) file and learn how to compile Falco from source [here](https://falco.org/docs/source).
2. Please label this pull request according to what type of issue you are addressing.
3. Please add a release note!
4. If the PR is unfinished while opening it specify a wip in the title before the actual title, for example, "wip: my awesome feature"
-->

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

<!--
Please remove the leading whitespace before the `/kind <>` you uncommented.
-->

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area API-version

> /area build

> /area CI

/area driver-kmod

/area driver-bpf

/area driver-modern-bpf

> /area libscap-engine-bpf

> /area libscap-engine-gvisor

> /area libscap-engine-kmod

> /area libscap-engine-modern-bpf

> /area libscap-engine-nodriver

> /area libscap-engine-noop

> /area libscap-engine-source-plugin

/area libscap-engine-savefile

/area libscap

> /area libpman

/area libsinsp

/area tests

> /area proposals

<!--
Please remove the leading whitespace before the `/area <>` you uncommented.
-->

**Does this PR require a change in the driver versions?**

> /version driver-API-version-major

> /version driver-API-version-minor

> /version driver-API-version-patch

> /version driver-SCHEMA-version-major

/version driver-SCHEMA-version-minor

> /version driver-SCHEMA-version-patch

<!--
Please remove the leading whitespace before the `/version <>` you uncommented.
-->

**What this PR does / why we need it**:

This PR is part of https://github.com/falcosecurity/libs/pull/2068. Besides extending, it also adds scap files conversion tests for events related to the accept syscall.

**Which issue(s) this PR fixes**:

<!--
Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
If PR is `kind/failing-tests` please post the related issues/tests in a comment and do not use `Fixes`.
-->

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

<!--
If no, you have to do nothing.
If yes, a release note is required:
Delete `NONE` and enter your extended release note in the block below.
Please note, the release note follows the "conventional commit specification" (https://www.conventionalcommits.org/en/v1.0.0/):
For example: `fix: broken link`.
If the PR requires additional action from users switching to the new release, prepend the string "action required:".
For example, `action required: change the API interface of libscap`.
-->

```release-note
NONE
```

FedeDP reviewed (approved):
/approve

poiana commented:
LGTM label has been added.  <details>Git tree hash: 1c4820b2df7acbfefd9ea14ef21eab9a9700f78c</details>

poiana commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/falcosecurity/libs/pull/2221#" title="Author self-approved">ekoops</a>*, *<a href="https://github.com/falcosecurity/libs/pull/2221#pullrequestreview-2580440185" title="Approved">FedeDP</a>*

The full list of commands accepted by this bot can be found [here](https://prow.falco.org/command-help?repo=falcosecurity%2Flibs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/falcosecurity/libs/blob/master/OWNERS)~~ [FedeDP]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

FedeDP commented:
/hold
there are old bpf verifier issues.

poiana commented:
New changes are detected. LGTM label has been removed.

FedeDP commented:
I triggered kernel-testing workflow against this PR : https://github.com/falcosecurity/libs/actions/runs/13070126804
Let's wait for the results!

FedeDP commented:
Arm64:
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |            BPF-PROBE SCAP-OPEN             |            MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|--------------------------------------------|---------------------------------------------|
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-54-modern-bpf_scap-open) |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-414-bpf-probe_build) | [🟡](#oraclelinux-414-bpf-probe_scap-open) | [🟡](#oraclelinux-414-modern-bpf_scap-open) |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |

Amd64:
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |             BPF-PROBE SCAP-OPEN             |             MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|---------------------------------------------|----------------------------------------------|
| amazonlinux2-4.19    | 🟢              | 🟢         | 🟢             | 🟢                                     | [❌](#amazonlinux2-419-bpf-probe_scap-open) | [🟡](#amazonlinux2-419-modern-bpf_scap-open) |
| amazonlinux2-5.10    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| amazonlinux2-5.15    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#amazonlinux2-54-modern-bpf_scap-open)  |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| amazonlinux2023-6.1  | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| archlinux-6.0        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| archlinux-6.7        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| centos-3.10          | 🟢              | 🟢         | 🟢             | [🟡](#centos-310-bpf-probe_build)      | [🟡](#centos-310-bpf-probe_scap-open)       | [🟡](#centos-310-modern-bpf_scap-open)       |
| centos-4.18          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| centos-5.14          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| fedora-5.17          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| fedora-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| oraclelinux-3.10     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-310-bpf-probe_build) | [🟡](#oraclelinux-310-bpf-probe_scap-open)  | [🟡](#oraclelinux-310-modern-bpf_scap-open)  |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#oraclelinux-414-modern-bpf_scap-open)  |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| oraclelinux-5.4      | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#oraclelinux-54-modern-bpf_scap-open)   |
| ubuntu-4.15          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#ubuntu-415-modern-bpf_scap-open)       |
| ubuntu-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#ubuntu-58-modern-bpf_scap-open)        |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |


All clean!

FedeDP commented:
Moving to next milestone; no need to hurry for the current one. 
/milestone 0.22.0

incertum commented:
@ekoops if you want this could be rebased now once you get a chance and we can start the review soon!

ekoops commented:
Hi @incertum . As you can see from the kernel testing matrix, there's an issue with loading the patched version on `amazonlinux2-4.19`. I still couldn't have chances to investigate the problem :disappointed:

ekoops commented:
Hey! I rebased and extracted the commit related to accept events converter tests into a separate PR.
Moreover, I slightly modified the legacy bpf probe patch... Let's see if we obtain the green light from amazonlinux2-4.19 by triggering the kernel-testing workflow!

FedeDP commented:
New matrix:
Amd64
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |             BPF-PROBE SCAP-OPEN             |             MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|---------------------------------------------|----------------------------------------------|
| amazonlinux2-4.19    | 🟢              | 🟢         | 🟢             | 🟢                                     | [❌](#amazonlinux2-419-bpf-probe_scap-open) | [🟡](#amazonlinux2-419-modern-bpf_scap-open) |
| amazonlinux2-5.10    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| amazonlinux2-5.15    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#amazonlinux2-54-modern-bpf_scap-open)  |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| amazonlinux2023-6.1  | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| archlinux-6.0        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| archlinux-6.7        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| centos-3.10          | 🟢              | 🟢         | 🟢             | [🟡](#centos-310-bpf-probe_build)      | [🟡](#centos-310-bpf-probe_scap-open)       | [🟡](#centos-310-modern-bpf_scap-open)       |
| centos-4.18          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| centos-5.14          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| fedora-5.17          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| fedora-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| oraclelinux-3.10     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-310-bpf-probe_build) | [🟡](#oraclelinux-310-bpf-probe_scap-open)  | [🟡](#oraclelinux-310-modern-bpf_scap-open)  |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | 🟢                                     | [❌](#oraclelinux-414-bpf-probe_scap-open)  | [🟡](#oraclelinux-414-modern-bpf_scap-open)  |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |
| oraclelinux-5.4      | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#oraclelinux-54-modern-bpf_scap-open)   |
| ubuntu-4.15          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#ubuntu-415-modern-bpf_scap-open)       |
| ubuntu-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | [🟡](#ubuntu-58-modern-bpf_scap-open)        |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                          | 🟢                                           |

Arm64:
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |            BPF-PROBE SCAP-OPEN             |            MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|--------------------------------------------|---------------------------------------------|
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-54-modern-bpf_scap-open) |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-414-bpf-probe_build) | [🟡](#oraclelinux-414-bpf-probe_scap-open) | [🟡](#oraclelinux-414-modern-bpf_scap-open) |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |

No luck :/

FedeDP commented:
arm64:
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |            BPF-PROBE SCAP-OPEN             |            MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|--------------------------------------------|---------------------------------------------|
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-54-modern-bpf_scap-open) |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-414-bpf-probe_build) | [🟡](#oraclelinux-414-bpf-probe_scap-open) | [🟡](#oraclelinux-414-modern-bpf_scap-open) |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢               

AMD64:
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |            BPF-PROBE SCAP-OPEN             |             MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|--------------------------------------------|----------------------------------------------|
| amazonlinux2-4.19    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-419-modern-bpf_scap-open) |
| amazonlinux2-5.10    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| amazonlinux2-5.15    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-54-modern-bpf_scap-open)  |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| amazonlinux2023-6.1  | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| archlinux-6.0        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| archlinux-6.7        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| centos-3.10          | 🟢              | 🟢         | 🟢             | [🟡](#centos-310-bpf-probe_build)      | [🟡](#centos-310-bpf-probe_scap-open)      | [🟡](#centos-310-modern-bpf_scap-open)       |
| centos-4.18          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| centos-5.14          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| fedora-5.17          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| fedora-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| oraclelinux-3.10     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-310-bpf-probe_build) | [🟡](#oraclelinux-310-bpf-probe_scap-open) | [🟡](#oraclelinux-310-modern-bpf_scap-open)  |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#oraclelinux-414-modern-bpf_scap-open)  |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| oraclelinux-5.4      | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#oraclelinux-54-modern-bpf_scap-open)   |
| ubuntu-4.15          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#ubuntu-415-modern-bpf_scap-open)       |
| ubuntu-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#ubuntu-58-modern-bpf_scap-open)        |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |

poiana commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/falcosecurity/libs/pull/2221#" title="Author self-approved">ekoops</a>*, *<a href="https://github.com/falcosecurity/libs/pull/2221#pullrequestreview-2867950266" title="Approved">FedeDP</a>*

The full list of commands accepted by this bot can be found [here](https://prow.falco.org/command-help?repo=falcosecurity%2Flibs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/falcosecurity/libs/blob/master/OWNERS)~~ [FedeDP]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

poiana commented:
LGTM label has been added.  <details>Git tree hash: 05a20b3169ac61fd12ad064d0ed6af3fcebb7438</details>

FedeDP commented:
Triggered once again kernel testing: https://github.com/falcosecurity/libs/actions/runs/15253335246
/hold

FedeDP commented:
amd64: 
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |            BPF-PROBE SCAP-OPEN             |             MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|--------------------------------------------|----------------------------------------------|
| amazonlinux2-4.19    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-419-modern-bpf_scap-open) |
| amazonlinux2-5.10    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| amazonlinux2-5.15    | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-54-modern-bpf_scap-open)  |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| amazonlinux2023-6.1  | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| archlinux-6.0        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| archlinux-6.7        | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| centos-3.10          | 🟢              | 🟢         | 🟢             | [🟡](#centos-310-bpf-probe_build)      | [🟡](#centos-310-bpf-probe_scap-open)      | [🟡](#centos-310-modern-bpf_scap-open)       |
| centos-4.18          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| centos-5.14          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| fedora-5.17          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| fedora-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| oraclelinux-3.10     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-310-bpf-probe_build) | [🟡](#oraclelinux-310-bpf-probe_scap-open) | [🟡](#oraclelinux-310-modern-bpf_scap-open)  |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#oraclelinux-414-modern-bpf_scap-open)  |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |
| oraclelinux-5.4      | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#oraclelinux-54-modern-bpf_scap-open)   |
| ubuntu-4.15          | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#ubuntu-415-modern-bpf_scap-open)       |
| ubuntu-5.8           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#ubuntu-58-modern-bpf_scap-open)        |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                           |


arm64:
|        KERNEL        | CMAKE-CONFIGURE | KMOD BUILD | KMOD SCAP-OPEN |            BPF-PROBE BUILD             |            BPF-PROBE SCAP-OPEN             |            MODERN-BPF SCAP-OPEN             |
|----------------------|-----------------|------------|----------------|----------------------------------------|--------------------------------------------|---------------------------------------------|
| amazonlinux2-5.4     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | [🟡](#amazonlinux2-54-modern-bpf_scap-open) |
| amazonlinux2022-5.15 | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| fedora-6.2           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| oraclelinux-4.14     | 🟢              | 🟢         | 🟢             | [🟡](#oraclelinux-414-bpf-probe_build) | [🟡](#oraclelinux-414-bpf-probe_scap-open) | [🟡](#oraclelinux-414-modern-bpf_scap-open) |
| oraclelinux-5.15     | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |
| ubuntu-6.5           | 🟢              | 🟢         | 🟢             | 🟢                                     | 🟢                                         | 🟢                                          |


let's go!
/unhold
